src/layouts/shared/page-header.module.css

```diff
@@ -56,3 +57,8 @@
 .pageHeader__logoLink:active {
   opacity: 0.8;
 }
+
+.pageHeader__link {
```

Comment:
Need to add a hover-state for this class. We don't have an "accent 0 hover" color, but try with `opacity: 0.75` or something similar to begin with (test in both dark and light mode for FRAM).